### ISSUE TITLE
Missing __version__ attribute

### ISSUE THREAD

**Issue by rasbt** [ResidentMario/missingno#22]
Hi there,
I just noticed that missingno does not import the `__version__` attribute properly, which causes the following problem:

```
>>> import missingno
>>> missingno.__version__
```

```
---------------------------------------------------------------------------
AttributeError                            Traceback (most recent call last)
<ipython-input-5-dc94dfc5cf5c> in <module>()
      1 import missingno
      2 
----> 3 missingno.__version__

AttributeError: module 'missingno' has no attribute '__version__'
```

Not sure how you think about it, but I think it would be useful to move the `__version__` attribute to `__init__` so that users can import and check the `__version__` of the missingno package that they are currently using.

**Comment by ResidentMario** [comment]
Should be (re)fixed now in the `0.3.4` release, cf. watermark#27, #19, my own clumsiness.

**Comment by rasbt** [comment]
Wow, that was quick, thanks! :)
